src/shared_modules/utils/windowsHelper.h

```diff
@@ -136,6 +169,77 @@ namespace Utils
         return ret;
     }
 
+
+    /* Reference: https://www.dmtf.org/sites/default/files/standards/documents/DSP0134_2.6.0.pdf */
+    static std::string parseRawSmbios(const BYTE* rawData, const DWORD rawDataSize)
```

Comment:
If I got this right, this function extracts the serial number out of the raw data of the SMBIOS. May be change its name to getSerialNumberFromSmbios or similar?